tinyquery/runtime.py

```diff
@@ -558,6 +558,24 @@ def _evaluate(self, num_rows, column):
                               values=values)
 
 
+class LastFunction(AggregateFunction):
+    def check_types(self, rep_list_type):
+        return rep_list_type
+
+    def _evaluate(self, num_rows, column):
+        values = []
+        if len(column.values) == 0:
+            values = [None]
+
+        if column.mode == tq_modes.REPEATED:
+            values = [repeated_row[-1] if len(repeated_row) > 0 else None
```

Comment:
Is it possible for column.values to be empty in repeated mode?  If so, I think this comprehension will die because `repeated_row` will be `None` which you can't take the len of.

Assuming that situation is possible, can you add a test-case for it?